exo-setup/features/setup.feature

```diff
@@ -39,3 +39,70 @@ Feature: Setup of Exosphere applications
     And ExoCom uses this routing:
       | ROLE             | SENDS                       | RECEIVES                    | NAMESPACE |
       | external-service | users.list, users.create    | users.listed, users.created |           |
+
+  Scenario: set up an application with services shared dependencies
```

Comment:
I think this test should go in `exosphere-shared` where `docker-compose.yml` is unit tested. This is a cucumber test which is related to more of what the user should be concerned about. Since the `docker-compose.yml` file shouldn't be modified by them anyways (hence why it lives in `tmp/`), then we shouldn't be testing it via cucumber.